.yamllint

```diff
@@ -0,0 +1,4 @@
+---
+extends: default
```

Comment:
I just noticed you snuck this in haha

Reply:
It's what was already running, just explicitly stated with a config now. Also removed the truthy test because of "on" in Travis

Reply:
Do you want the config deleted?

Reply:
Moved to new branch